src/components/SVGIcon/icons/expand.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M20 12.4444V18.2222C20 19.2041 19.2041 20 18.2222 20H12.4444C12.199 20 12 19.801 12 19.5556C12 19.3101 12.199 19.1111 12.4444 19.1111H18.2222C18.7131 19.1111 19.1111 18.7131 19.1111 18.2222V12.4444C19.1111 12.199 19.3101 12 19.5556 12C19.801 12 20 12.199 20 12.4444Z" fill="#757575"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14.2169 10.416C14.0431 10.5898 13.7613 10.5898 13.5876 10.416C13.4138 10.2422 13.4138 9.96045 13.5876 9.78667L18.4889 4.88889H15.6267C15.5097 4.88843 15.3979 4.84117 15.316 4.75766C15.2342 4.67416 15.1892 4.56135 15.1911 4.44445C15.1911 4.20242 15.3847 4.00484 15.6267 4H19.5671C19.6825 3.9996 19.7932 4.04537 19.8747 4.12711C19.9548 4.20981 19.9998 4.32039 20 4.43556V8.37689C19.9995 8.49381 19.9523 8.60568 19.8688 8.68752C19.7853 8.76936 19.6725 8.81435 19.5556 8.81245C19.3135 8.8125 19.1159 8.61887 19.1111 8.37689V5.51734L14.2169 10.416Z" fill="#757575"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9.78667 13.5874C9.96045 13.4136 10.2422 13.4136 10.416 13.5874C10.5898 13.7612 10.5898 14.0429 10.416 14.2167L5.51733 19.1109H8.37689C8.49381 19.1114 8.60568 19.1586 8.68752 19.2422C8.76936 19.3257 8.81435 19.4385 8.81244 19.5554C8.81249 19.7974 8.61887 19.995 8.37689 19.9998H4.43556C4.32046 20.0001 4.21005 19.9543 4.12889 19.8727C4.04806 19.7912 4.00187 19.6817 4 19.5669V15.6265C4.00046 15.5096 4.04772 15.3977 4.13123 15.3159C4.21474 15.234 4.32754 15.189 4.44444 15.1909C4.68647 15.1909 4.88405 15.3845 4.88889 15.6265V18.4887L9.78667 13.5874Z" fill="#757575"/>
```

Comment:
we generally avoid hard-coded fills with fill rules and clip rules.. have we verified that this is usable in the same way as the other icons? 🤔

Reply:
https://github.com/instacart/Snacks/blob/master/src/components/SVGIcon/SVGIcon.js#L21

i don't think the `fill` passed here is going to work with these icons

Reply:
tbh, I just added the resources as handed over from design without checking-- I can't get the styleguide to compile locally. Any chance we can quickly pair on this so I know what to look for?

Reply:
<img width="110" alt="Screen Shot 2019-12-09 at 10 00 43 AM" src="https://user-images.githubusercontent.com/1967219/70460570-85fd1500-1a6b-11ea-9e4e-31200e84b6e6.png">

Updated!